week-06-reflections/anguyen82-stat660.md

```diff
@@ -2,8 +2,94 @@
 # Questions about Assigned Quiz Problems and Recipes
 
 
+[Chapter 10 Quiz, Problem 1]
+*Question (anguyen82-stat660): What kind of data combining method is being used in this question?
```

Comment:
Please insert spaces after asterisks, so that bulleted lists properly render:

```suggestion
* Question (anguyen82-stat660): What kind of data combining method is being used in this question?
```